webapp/src/auth.ts

```diff
@@ -343,14 +349,23 @@ type ApiResult<T> = {
     errmsg: string;
 };
 
+const BACKEND_DEFAULT = "";
+const BACKEND_PROD = "https://www.makecode.com";
```

Comment:
to be friendly to OSS, a 3rd party should be able to override this. Our API used ``/api/`` to allow this.

Reply:
(something to ponder about)

Reply:
In the prod environment the url is relative and will hit whatever backend the site was served from. This is a dev-only override.

Reply:
I scoped it to only apply when running on localhost/dev.
https://github.com/microsoft/pxt/pull/7604/files/ca9311a75f5f0aa58f911043a42544b2708ae7c3..e5bea850a769873bda22234bc26f4c967a27fc34